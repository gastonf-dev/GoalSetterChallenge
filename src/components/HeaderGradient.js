/**
 * @flow stric-local
 * @format
 */


import React from 'react';
import type {Node} from 'react';

import {StyleSheet, Text, View} from 'react-native';
import LinearGradient from 'react-native-linear-gradient';

const HeaderGradient = (): Node => {
  return (
    <LinearGradient
      colors={['#8D24C4', '#500F71']}
      style={{flex: 1}}
      start={{x: 0, y: 0}}
      end={{x: 1, y: 0}}
    />
  );
};

export default HeaderGradient;

const styles = StyleSheet.create({});
