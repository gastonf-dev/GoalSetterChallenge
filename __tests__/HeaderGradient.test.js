import React from 'react';
import renderer from 'react-test-renderer';
import HeaderGradient from '../src/components/HeaderGradient';

jest.mock('react-native-linear-gradient', () => 'LinearGradient');

describe('<HeaderGradient />', () => {
  test('render header gradient', () => {
    const wrapper = renderer.create(<HeaderGradient />);
    expect(wrapper).toMatchSnapshot();
  });
});
