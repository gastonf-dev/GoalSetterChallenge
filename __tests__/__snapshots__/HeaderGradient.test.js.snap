// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderGradient /> render header gradient 1`] = `
<LinearGradient
  colors={
    Array [
      "#8D24C4",
      "#500F71",
    ]
  }
  end={
    Object {
      "x": 1,
      "y": 0,
    }
  }
  start={
    Object {
      "x": 0,
      "y": 0,
    }
  }
  style={
    Object {
      "flex": 1,
    }
  }
/>
`;
